### PR TITLE
Ai polish

### PR DESCRIPTION
**Pull request by DantElemer**
Noone can beat my AIs from now on!

### CODE REVIEW

File: src/Client/log.txt [hunk 1]
```diff
@@ -1,10 +1,7 @@
...
-2020-05-04 10:27:07,310 - Imigyen sz�la Domi.Screen: - INFO - ID: 1 Death of player acknowledged.
-2020-05-04 10:27:07,312 - Imigyen sz�la Domi.Screen: - INFO - ID: 1 Player removed from player manager.
-2020-05-04 10:27:09,141 - Imigyen sz�la Domi.ClientCommunicator: - INFO - Client socket closed.
+2020-05-05 11:07:39,165 - Imigyen sz�la Domi.ClientCommunicator: - INFO - Successful connection!
+2020-05-05 11:07:39,168 - Imigyen sz�la Domi.Client: - INFO - I got my id: 1
+2020-05-05 11:07:39,194 - Imigyen sz�la Domi.Screen: - INFO - Game started, terrain loaded
+2020-05-05 11:08:00,728 - Imigyen sz�la Domi.Screen: - INFO - ID: 2 Death of p2020-05-05 11:08:00,734 - Imigyen sz�la Domi.Screen: - INFO - ID: 2 Death of player acknowledged.
+2020-05-05 11:08:00,734 - I2020-05-05 11:08:36,261 - Imigyen sz�la Domi.Screen: - INFO - ID: 3 Death of player acknowledged.
+2020-05-05 11:08:36,261 - Imigyen sz�la Domi.Screen: - INFO - ID: 3 Player removed from player manager.
+migyen sz�la Domi.Screen: - INFO - ID: 3 Player removed from player manager.
```
Comment (marzsolt) [inline src/Client/log.txt:7]: .gitignore not working?

File: src/Server/Game/Game.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def __init__(self):
         self.logger = logging.getLogger('Domi.Game')
         self.__game_started = False
         self.__chose_host = False
-        self.__AI_number = 2
+        self.__AI_number = 5
         self.__human_player_number = 2  # remember to adjust this default with screen's first player's selector's
         self.__first_player_id = None
```
Comment (marzsolt) [inline src/Server/Game/Game.py:27]: That's a lot, I want back 2!

File: src/Server/Player/PlayerLogic.py [hunk 3]
```diff
@@ -38,10 +39,11 @@ def __init__(self, player_id, terrain, game):
...
         self._is_flying = True
-        self._can_attack = True
+        self._attack_on_cooldown = False
         self._is_attacking = False
         self.can_get_hurt = True
-        self.hp = 100
+        self.max_hp = 100
+        self.hp = self.max_hp
 
     def update(self):
```
Comment (marzsolt) [inline src/Server/Player/PlayerLogic.py:48]: That max_hp sounds like a constant. Poor him is crying to be treated correspondingly.

File: src/Server/Player/PlayerLogic.py [hunk 4]
```diff
@@ -80,24 +82,25 @@ def _send_updated_pos_hp(self):
...
+        return not self._attack_on_cooldown
+
     def _attack(self):
-        #if self._can_attack:
         self._is_attacking = True
-        self._can_attack = False
+        self._attack_on_cooldown = True
         self.can_get_hurt = False
-        Timer.sch_fun(100, self.restore_attackaibility, ())
+        Timer.sch_fun(100, self.restore_attack_ability, ())  # TODO use constant/var!
```
Comment (marzsolt) [inline src/Server/Player/PlayerLogic.py:96]: Remember to remove the TODO when merging, as we should get rid of them.

File: src/Client/log.txt (hunk 1 above)
Comment (DantElemer) [reply src/Client/log.txt:7]: Not really... as far as I know if a file is "tracked" by git, it won't release it :( If we delete it and commit the deletion maybe after that gitignore can work.
I try to discard it usually, but it seems not always...

File: src/Server/Game/Game.py (hunk 2 above)
Comment (DantElemer) [reply src/Server/Game/Game.py:27]: Actually my local version has 6 :P 
It's fun if there are many in the party! :)

File: src/Server/Player/PlayerLogic.py (hunk 3 above)
Comment (DantElemer) [reply src/Server/Player/PlayerLogic.py:48]: Python somewhy hates those big letters :( apart from that if someone drinks super sugar juice he may increase his full hp :)

File: src/Server/Player/PlayerLogic.py (hunk 4 above)
Comment (DantElemer) [reply src/Server/Player/PlayerLogic.py]: Good catch! I made a commit to do that, but I'm not sure what git thinks now :(

**Pull request merged**